README.md

```diff
@@ -1 +1,45 @@
-# weatherman_lab
\ No newline at end of file
+# Weatherman Lab
+
+## Steps to run the project
+
+  First get a clone of the repo
```

Comment:
When you write readme.md always add example command like 

First get a clone of the repo 
`git clone <this_repo_link>`